core/src/main/scala/caliban/execution/Executor.scala

```diff
@@ -101,42 +101,63 @@ object Executor {
 
     def makeQuery(step: ReducedStep[R], errors: Ref[List[CalibanError]]): ZQuery[R, Nothing, ResponseValue] = {
 
+      def eraseError(error: CalibanError): UQuery[ResponseValue] =
```

Comment:
Maybe name is `handleError` or something like that since "erase" sounds like we just throw it away.

Reply:
I tried but wrapper returns a `CalibanError`, should I make that return `ExecutionError` too?

Reply:
That's a fair point we can all the downstream wrappers to clean up errors from the upstream.